Makefile

```diff
@@ -14,6 +14,15 @@ proto: proto-builder
 		$(PROTO_DOCKER_IMAGE) \
 		protoc -I/proto -I./api --go_out=plugins=grpc:./pkg/api/validator ./api/validator.proto
 
+.PHONY: proto
```

Comment:
Typo?

```suggestion
.PHONY: pyproto
```

Reply:
Do we need to add `make pyproto` to the [README.md](https://github.com/forseti-security/config-validator/blob/master/README.md)?

Reply:
Yes good idea, cc @briantkennedy

Reply:
done